web_widget_color/static/src/js/widget.js

```diff
@@ -71,15 +71,11 @@ odoo.define('web.web_widget_color', function(require) {
     /*
      * Init jscolor for each editable mode on view form
      */
-    FormView.include({
-        on_button_edit: function () {
+       FormView.include({
+        on_form_changed: function () {
             this._super();
             jscolor.init(this.$el[0]);
```

Comment:
jscolor is not correctly used in this module.
init doesn't accept any parameter... when you call init, jscolor initialize all elements with 'color' class.

Line 61 is wrong too...
See https://github.com/OCA/web/blob/10.0/web_widget_color/static/lib/jscolor/jscolor.js#L27